src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -86593,5 +86593,55 @@ public void M(TypedReference r)
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "__refvalue(r, string?)").WithLocation(7, 13)
                 );
         }
+
+        [Fact, WorkItem(25375, "https://github.com/dotnet/roslyn/issues/25375")]
+        public void ParamsNullable_SingleNullParam()
+        {
+            var source =
+@"
+class C
+{
+    static void F(object x, params object?[] y)
+    {
+    }
+
+    static void G(object x, params object[]? y)
+    {
+    }
+
+    static void Main()
+    {
+        // Both calls here are invoked in normal form, not expanded
+
+        F("""", null); // 1
+        G("""", null);
+
+        // These are called with expanded form
+
+        F("""", null, """"); 
+        G("""", null, """"); // 2
+
+        // Explicitly called with array
+
+        F("""", new object?[] { null }); 
+        G("""", new object[] { null }); // 3
```

Comment:
is the behavior the same with `G("""", new object?[] { null });`?

Reply:
You end up with 'Argument of type 'object?[]' cannot be used for parameter 'y' of type 'object[]'' , but we have plenty of tests to cover that. I mainly wanted to show the explicit array calls for completeness.

---
In reply to: [267460029](https://github.com/dotnet/roslyn/pull/34293#discussion_r267460029) [](ancestors = 267460029)